### DISCUSSION TITLE
Better search engine layout

### DISCUSSION THREAD

**Discussion by TheIJM** [VueTorrent/VueTorrent#2022] [Ideas]
I’d love to see an improved design for the search engine, ideally something more in line with the torrent list layout. Since I mostly use it on my phone, it would be helpful to have more information displayed effectively in vertical orientation. The torrent list has a clear separation of elements, which makes it easier to read compared to the current search engine layout.

P.S. 
It would also be great if the sidebar didn’t open automatically when I switch to landscape mode. This might not be necessary if the search engine layout were more compact, allowing us to see multiple results at once in vertical orientation.

**Comment by Larsluph** [comment]
Is the problem only linked to mobile layout or do you also have issues on desktop?

Another thread has been created a few days ago #1980, do you agree with the proposed layout?
